packages/nimble-components/src/spinner/specs/README.md

```diff
@@ -0,0 +1,146 @@
+(TBD)
+
+## Implementation
+
+Component implementation will derive from the FAST `FoundationElement`.
+
+**Alternatives Considered:** We could derive from `fast-progress-ring` (radial progress indicator, which supports an indeterminate progress display), however then our component API would also pick up the `min`/`max`/`value` properties for determinate progress (and `paused` to pause the indicator), which we don't plan on supporting at this time. Additionally, the visual appearance of our spinner will be a little different from a radial progress bar, and we still may end up supporting a standard radial progress bar in the future.  
+For reference, the FAST spec for `progress` / `progress-ring` is [here](https://github.com/microsoft/fast/blob/802443ffb2b19a078f9b48f62e6d1a35e3276fb5/packages/web-components/fast-foundation/src/progress/README.md).
+
+### States
+
+Only one component state: visible and animating. Pausing not supported; spinner is expected to be added/removed from the DOM as needed.
+
+### Accessibility
+
+No keyboard states / form integration.
+
+ARIA: We plan to use the [`progressbar role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role) without setting `aria-valuemin` / `aria-valuemax` / `aria-valuenow` (representing indeterminate).  
+This matches the behavior of an indeterminate FAST progress ring (in which case min/max/value are unset) - see the [FAST template here](https://github.com/microsoft/fast/blob/802443ffb2b19a078f9b48f62e6d1a35e3276fb5/packages/web-components/fast-foundation/src/progress-ring/progress-ring.template.ts#L17).
+
+`prefers-reduced-motion`: A simplified spinner indicator will be shown when this is enabled, see the Visual Appearance section.
+
+### Globalization
+
+N/A.
+
+RTL: No special handling / no changes to the spinner animation. Linear indeterminate progress animations are sometimes reversed for RTL, but circular/clockwise ones are usually not (see [Material Design v2, Mirroring Elements](https://m2.material.io/design/usability/bidirectionality.html#mirroring-elements) > _When Not To Mirror_, for one example).
+
+### Security
+
+N/A
+
+### Performance
+
+N/A
+
+### Dependencies
+
+None
+
+### Test Plan
+
+Standard tests planned.  
+[Chromatic is supposed to handle pausing CSS animations by default](https://www.chromatic.com/docs/animations), however we should verify that by ensuring the snapshots of the spinner show the same state across 2 Chromatic builds.
+
+### Tooling
+
+N/A
+
+### Documentation
+
+N/A - Standard updates of the component table in the README, and in Storybook.
+
+## Open Issues
+
+-   Decide if we want to require the `size` attribute. (We could scale to user-provided sizes but they may end up with non-optimal bit sizes and padding/margins not picked by a designer)
+-   Decide if we need the `theme-variant` (PowerGreen variant for color theme). Brandon O'Keefe to follow up on this point.
```

Comment:
Tagging @NIbokeefe to follow up to give guidance on this open question (if we need the alt color / PowerGreen version)

Reply:
Yea I think the question is whether we just need the PowerGreen version (so we can make an attribute to make it consistent in all the use-cases) or if we more generally need a way for apps to specify a custom color (in which case may instead copy the the icon pattern and create a token like `spinner-color` that [can be overridden](https://github.com/ni/nimble/blob/95cbaea9c0ae16b804816ac68204253735bc7b84/packages/nimble-components/src/icon-base/styles.ts#L44) for specific use cases)

Reply:
Split this discussion/ open question off into #916 .

For the purposes of this spec, and for the initial implementation, we will omit supporting the 'Color UI Alt' (PowerGreen) variant. Added notes about this to the spec in a new *Potential Future Enhancements* section, and moved the content about `theme-variant` there too. (`theme-variant` will not be in the initial implementation)